### DISCUSSION TITLE
MIT License on SourceForce

### DISCUSSION THREAD

**Discussion by obonilla66** [LoboEvolution/LoboEvolution#98] [General]
In think that the change of GPL to MIT License would raise the interest of developers.
But in Sourceforce keeps indicating that the license is GPL, ruining the license change.
It is possible to change it?
Regards.

**Comment by oswetto** [comment]
Hello @obonilla66 from LoboEvolution 4.0 the license is MIT https://github.com/LoboEvolution/LoboEvolution/blob/master/LICENSE
